### ISSUE TITLE
Error just after installing from playstore related to firebase

### ISSUE THREAD

**Issue by rmallah** [nextcloud/android#7616]
```
************ CAUSE OF ERROR ************

java.lang.RuntimeException: Unable to get provider com.google.firebase.provider.FirebaseInitProvider: java.lang.IllegalArgumentException: Please set your project ID. A valid Firebase project ID is required to communicate with Firebase server APIs: It identifies your project with Google.
	at android.app.ActivityThread.installProvider(ActivityThread.java:6329)
	at android.app.ActivityThread.installContentProviders(ActivityThread.java:5918)
	at android.app.ActivityThread.handleBindApplication(ActivityThread.java:5857)
	at android.app.ActivityThread.-wrap3(ActivityThread.java)
	at android.app.ActivityThread$H.handleMessage(ActivityThread.java:1699)
	at android.os.Handler.dispatchMessage(Handler.java:102)
	at android.os.Looper.loop(Looper.java:154)
	at android.app.ActivityThread.main(ActivityThread.java:6682)
	at java.lang.reflect.Method.invoke(Native Method)
	at com.android.internal.os.ZygoteInit$MethodAndArgsCaller.run(ZygoteInit.java:1520)
	at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:1410)
Caused by: java.lang.IllegalArgumentException: Please set your project ID. A valid Firebase project ID is required to communicate with Firebase server APIs: It identifies your project with Google.
	at com.google.android.gms.common.internal.Preconditions.checkNotEmpty(Unknown Source)
	at com.google.firebase.iid.FirebaseInstanceId.checkRequiredFirebaseOptions(com.google.firebase:firebase-iid@@20.2.3:52)
	at com.google.firebase.iid.FirebaseInstanceId.getInstance(com.google.firebase:firebase-iid@@20.2.3:2)
	at com.google.firebase.iid.FirebaseInstanceId.getInstance(com.google.firebase:firebase-iid@@20.2.3:1)
	at com.google.firebase.iid.Store.checkForRestore(com.google.firebase:firebase-iid@@20.2.3:15)
	at com.google.firebase.iid.Store.<init>(com.google.firebase:firebase-iid@@20.2.3:5)
	at com.google.firebase.iid.FirebaseInstanceId.<init>(com.google.firebase:firebase-iid@@20.2.3:16)
	at com.google.firebase.iid.FirebaseInstanceId.<init>(com.google.firebase:firebase-iid@@20.2.3:8)
	at com.google.firebase.iid.Registrar.lambda$getComponents$0$Registrar(com.google.firebase:firebase-iid@@20.2.3:26)
	at com.google.firebase.iid.Registrar$$Lambda$0.create(com.google.firebase:firebase-iid@@20.2.3)
	at com.google.firebase.components.ComponentRuntime.lambda$new$0(com.google.firebase:firebase-components@@16.0.0:69)
	at com.google.firebase.components.ComponentRuntime$$Lambda$1.get(com.google.firebase:firebase-components@@16.0.0)
	at com.google.firebase.components.Lazy.get(com.google.firebase:firebase-components@@16.0.0:53)
	at com.google.firebase.components.ComponentRuntime.initializeEagerComponents(com.google.firebase:firebase-components@@16.0.0:158)
	at com.google.firebase.FirebaseApp.initializeAllApis(com.google.firebase:firebase-common@@19.3.0:563)
	at com.google.firebase.FirebaseApp.initializeApp(com.google.firebase:firebase-common@@19.3.0:304)
	at com.google.firebase.FirebaseApp.initializeApp(com.google.firebase:firebase-common@@19.3.0:268)
	at com.google.firebase.FirebaseApp.initializeApp(com.google.firebase:firebase-common@@19.3.0:253)
	at com.google.firebase.provider.FirebaseInitProvider.onCreate(com.google.firebase:firebase-common@@19.3.0:51)
	at android.content.ContentProvider.attachInfo(ContentProvider.java:1758)
	at android.content.ContentProvider.attachInfo(ContentProvider.java:1733)
	at com.google.firebase.provider.FirebaseInitProvider.attachInfo(com.google.firebase:firebase-common@@19.3.0:45)
	at android.app.ActivityThread.installProvider(ActivityThread.java:6326)
	... 10 more

************ APP INFORMATION ************
ID: com.nextcloud.client
Version: 30140190
Build flavor: gplay

************ DEVICE INFORMATION ************
Brand: samsung
Device: noblelte
Model: SM-N9208
Id: NRD90M
Product: nobleltextc

************ FIRMWARE ************
SDK: 24
Release: 7.0
Incremental: N9208XXS5CRH2
```

**Comment by Cthanatos** [comment]
I got the same error this morning, on a Galaxy Note 8. I logged into github to check for another issue I'm experiencing, not being able to upload multiple items at once, and saw this. I figured I'd corroborate the issue at least.

**Comment by rmallah** [comment]
I could not upload single items also. So i thought i would un-install and install again but bumped into a new issue. Hope its fixed soon , mobile app is an extremely significant part of the nextcloud ecosystem/offering.

**Comment by AndyScherzinger** [comment]
@tobiasKaminsky see above, this has been reported several times

**Comment by tobiasKaminsky** [comment]
Does this only happen on fresh installations?

**Comment by BernhardGehl** [comment]
> Does this only happen on fresh installations?

Browsing through the issues for notification-related open issues, I found this. In fact my device (European Samsung S10e stock Android 10) seems to get this after every fresh install / data reset of the app. 
* The rest of the devices in my household (2x Samsung A40, 1x Samsung A20e, 1x Samsung A3, 1x Motorola G Pro, all stock OEM ROMs) are not affected.
* Probably related: I don't get push notifications but when I look in the app, it gets notifications
* Probably not related (I might file a separate issue): The contents of some folders don't show up in the app - the loading spinner goes on forever. (As I said, probably a different thing)

**Comment by genuineBuildMonkey** [comment]
Same issue. Samsung Galaxy S8.

**Comment by tobiasKaminsky** [comment]
I added the missing config option.
Can you please test this: https://cloud.nextcloud.com/s/Xd74wxsT2PMxn3f

It is signed with the same key as 3.15.0, but internally it is 3.16 Alpha.
So you can upgrade it without any problems.

**Comment by joshtrichards** [comment]
Fixed in #10241
